### DISCUSSION TITLE
Encoder weights not frozen after phase 1 with discrete configuration

### DISCUSSION THREAD

**Discussion by snnithya** [acids-ircam/RAVE#246] [Q&A]
Today I realised that the encoder weights aren't being frozen after phase 1 of training in the [DiscreteEncoder module](https://github.com/acids-ircam/RAVE/blob/master/rave/blocks.py) (used by discrete.gin config) and gradients are still being backpropogated through the encoder in phase 2 of training as well. I was wondering if this was intentional and if so why? I'm a bit confused by this since according to the original [RAVE article](https://arxiv.org/abs/2111.05011), the encoder is supposed to be 'frozen' during phase 2 of training. Thanks in advance!
